docs/hsi.xml

```diff
@@ -281,6 +281,34 @@
       </note>
     </refsect3>
 
+    <refsect3 id="org.fwupd.hsi.Uefi.PK.TestKey">
```

Comment:
I think it's better to label this as "unsecure key".  The logic behind it being that if any evidence of another platform key that is not properly secured, even if not intended for testing was discovered it could be added to this as a disallow list.

Reply:
Good idea. Unsecure or insecure? I know there's a difference, but the first looks weird.

Reply:
Actually, all the HSI keys are in the "positive" so it needs to be `NotUnsecureKey` or maybe just `org.fwupd.hsi.Uefi.PK.Valid`

Reply:
Maybe just `org.fwupd.hsi.Uefi.PK` too; with states "valid" or "invalid"

Reply:
Don't have a strong feeling between your last 3 comments.  Whichever sounds best to you is fine.

Reply:
I went with the latter...